competitions/s1floods/benchmark_tutorial.ipynb

```diff
@@ -0,0 +1,2029 @@
+    "\n",
+    "The training data for this competition consist of 542 \"chips\" from flood events around the world, which each contain two polarization bands. Each chip has a corresponding label mask that indicates which pixels in a scene contain water. You can optionally supplement this training data with information on permanent water sources and elevation, available through the [Microsoft AI for Earth STAC API](https://github.com/radiantearth/stac-api-spec) (see the competition [STAC resources](https://www.drivendata.org/competitions/81/detect-flood-water/page/388/) page for more information on bringing in this data). To enter this challenge, you will package and submit your code to generate predictions as single-band tifs with binary pixel values.\n",
+    "\n",
+    "In this post, we'll cover:\n",
+    "\n",
+    "- [Exploring the data](#section1)\n",
+    "- [Preparing the data](#section2)\n",
+    "- [Explaining the benchmark model](#section3)\n",
+    "- [Fitting the benchmark model](#section4)\n",
+    "- [Generating a submission in the correct format](#section5)\n",
+    "\n",
+    "Let's get started!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# If running on the Planetary Computer, uncomment below\n",
+    "# !pip install watermark"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a id=\"section1\"></a>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Python implementation: CPython\n",
+      "Python version       : 3.8.10\n",
+      "IPython version      : 7.26.0\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "%load_ext watermark\n",
+    "%watermark -v"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "%watermark -i"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Explore the Data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Metadata"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "First we'll download the training data to a new subdirectory called `training_data/` in our current working directory. If you are working on the Planetary Computer Hub, consider using [Azure Blob Storage](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blobs-introduction) to store your data using the West Europe region, which is where your server is running.\n",
```

Comment:
Can you add a link to https://planetarycomputer.microsoft.com/docs/quickstarts/storage/ as well. Something like

> See [Writing data to Azure Blob Storage](https://planetarycomputer.microsoft.com/docs/quickstarts/storage/) for an example.